userspace/chisel/chisel_api.cpp

```diff
@@ -194,6 +195,22 @@ uint32_t lua_cbacks::rawval_to_lua_stack(lua_State *ls, uint8_t* rawval, ppm_par
 				}
 			}
 			break;
+		case PT_CHARBUFARRAY:
```

Comment:
I'm not 100% confident that this is what we want here, but I think flattening the list to a string would work for chisels.